push_notifications/settings.py

```diff
@@ -38,8 +28,8 @@
 
 # WP (WebPush)
 PUSH_NOTIFICATIONS_SETTINGS.setdefault("WP_POST_URL", {
-	"CHROME": PUSH_NOTIFICATIONS_SETTINGS["FCM_POST_URL"],
-	"OPERA": PUSH_NOTIFICATIONS_SETTINGS["FCM_POST_URL"],
+	"CHROME": "https://fcm.googleapis.com/fcm/send",
+	"OPERA": "https://fcm.googleapis.com/fcm/send",
```

Comment:
Are we sure about those (referring to the todos on the same urls about the wns config)

Reply:
i dont know how web push works tbh. i reverted the change and left in the `FCM_POST_URL` so users that changed that will not have breaking changes to that part of the application